models/validator.js

```diff
@@ -511,6 +511,54 @@ const schemas = {
     return contentSchema;
   },
 
+  ad: function () {
+    let contentSchema = Joi.object({
+      children: Joi.array().optional().items(Joi.link('#content')),
+    })
+      .required()
+      .min(1)
+      .id('ad');
```

Comment:
Eu ainda me atrapalho com o Joi, então não sei ao certo, mas aqui não deveria ser um link com `#ad` ao invés de `#content`?

```suggestion
      children: Joi.array().optional().items(Joi.link('#ad')),
    })
      .required()
      .min(1)
      .id('ad');
```

Eu clonei a branch para testar eu mesmo, mas ou testei errado, ou não teve diferença.

Reply:
Os comentários seguem como `content`.

Como o model por enquanto só tem o `getRandom`, que não precisa devolver os comentários, então nem todos os campos desse esquema estão sendo utilizados por enquanto.

Reply:
Ah, entendi. O `children` me confundiu; pensei que tinha algo a ver com a lista de conteúdos, e não com a propriedade `children` presente no conteúdo :sweat_smile: